JSS/using_drtmle_jss.Rmd

```diff
@@ -1261,7 +1263,7 @@ riskRatio_1v0 <- list(f = function(eff){ log(eff) },
 ci(glm_fit_multiA, contrast = riskRatio_1v0)
 ```
 
-Just as straightforwardly, we can compute confidence intervals for the risk
+Similarly, we can compute confidence intervals for the risk
```

Comment:
I'm not a fan of "Just as straightforwardly" either, but note that you used "Similarly" just two sentences above. It's nice to mix it up every now and then 😎